ballsdex/packages/config/cog.py

```diff
@@ -128,3 +128,33 @@ async def disable(self, interaction: discord.Interaction["BallsDexBot"]):
                     f"{settings.bot_name} is now enabled in this server, however there is no "
                     "spawning channel set. Please configure one with `/config channel`."
                 )
+
+    @app_commands.command()
+    @app_commands.checks.has_permissions(manage_guild=True)
+    async def status(self, interaction: discord.Interaction["BallsDexBot"]):
+        """
+        Check the server configuration status.
+        """
+        config = await GuildConfig.get_or_none(guild_id=interaction.guild_id)
+        if not config or not config.spawn_channel:
+            await interaction.response.send_message(
+                f"{settings.bot_name} is not configured in this server yet.\n"
+                "Please use `/config channel` to set a channel.",
+                ephemeral=False,
+            )
+        else:
+            channel = interaction.guild.get_channel(config.spawn_channel)
```

Comment:
You can make the error go away by asserting that `interaction.guild` exists.
I also added a check for the guild's availability, very rare situation but worth handling. 
```suggestion
            assert interaction.guild
            if interaction.guild.unavailable:
                await interaction.response.send_message(
                    "Your server is unavailable to the bot. Readding it may fix this."
                )
                return
            channel = interaction.guild.get_channel(config.spawn_channel)
```